rc/filetype/scheme.kak

```diff
@@ -38,14 +38,15 @@ require-module lisp
 add-highlighter shared/scheme regions
 add-highlighter shared/scheme/code default-region group
 
-add-highlighter shared/scheme/string region '"' (?<!\\)(\\\\)*" fill string
-add-highlighter shared/scheme/comment region ';' '$' fill comment
+add-highlighter shared/scheme/string region %{(?<!#\\)"} (?<!\\)(\\\\)*" fill string
+add-highlighter shared/scheme/comment region %{(?<!#\\);} '$' fill comment
 add-highlighter shared/scheme/comment-form region -recurse "\(" "#;\(" "\)" fill comment
 add-highlighter shared/scheme/comment-block region "#\|" "\|#" fill comment
 add-highlighter shared/scheme/quoted-form region -recurse "\(" "'\(" "\)" fill variable
 
 add-highlighter shared/scheme/code/ regex (#t|#f) 0:value
 add-highlighter shared/scheme/code/ regex \b[0-9]+\.[0-9]*\b 0:value
+add-highlighter shared/scheme/code/ regex (#\\((\w+)|(.))) 0:value
```

Comment:
I think having the character names here would be nice, so that I don't try `#\nl` or something.

According to r6rs,
```
 <character name> → nul ∣ alarm ∣ backspace ∣ tab
    ∣ linefeed ∣ newline ∣ vtab ∣ page ∣ return
    ∣ esc ∣ space ∣ delete
```
(r5rs has just space and newline).

So, `(nul\b|alarm\b|backspace\b| ... |.)`?

Reply:
I couldn't get the highlights working with `((nul)|(alarm)|...)` (`...` standing for more items in the list). That's why I took this shortcut.

Also, I'm just noticing that instead of `.` I should use `\S`.

Reply:
Actually, `#\ ` is a valid way of saying `#\space`, weirdly enough.

Alternation works slightly differently in Kakoune, preferring the left-most alternative rather than the longest.  That's bitten me when making a list of words like '(foo|bar|baz)', especially if you put `.` first.  (The nested parens here should be unnecessary, though I guess shouldn't be a problem.)